libs/language-server/src/lib/builtin-library/jayvee-standard-library.ts

```diff
@@ -0,0 +1,9 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+export const STANDARD_LIBRARY_FILENAME = 'standard.jv';
+
+export const STANDARD_LIBRARY_SOURCECODE = `
+// TBD
```

Comment:
Some loose end here?

Reply:
No, the std library is supposed to be empty, so it contains just that comment on purpose.